src/pages/cart/main/Main.jsx

```diff
@@ -1,7 +1,10 @@
-import { carts } from "staticData/data";
+import { useCart } from "contexts/CartContext";
+import { Card } from "./Card";
 
-const Main = () => (
-  <>
+const Main = () => {
```

Comment:
Is this `Main` component used on multiple pages?

Reply:
NO